client/src/components/MemberList.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { useState } from 'react';
```

Comment:
These two lines can be combined:
```suggestion
import React, { useState } from 'react';
```